src/config/dynamic-theme-fixes.config

```
@@ -7476,6 +7476,13 @@ body
 
 ================================
 
+hmong.*/wiki/*
```

`hmong.*` Is sufficient enough
```suggestion
hmong.*
```